tests/clients/test_job.py

```diff
@@ -210,6 +265,31 @@ def test_job_summary_schema():
     assert data == JOB_SUMMARY
 
 
+def test_instance_size_schema():
+    data = InstanceSizeSchema().load(INSTANCE_SIZE_BODY)
+    assert data == INSTANCE_SIZE
+
+
+def test_job_parameter_schema():
+    data = JobParameterSchema().load(JOB_PARAMETERS_BODY)
+    assert data == JOB_PARAMETERS
+
+
+def test_job_command_schema():
+    data = JobCommandSchema().load(JOB_COMMAND_BODY)
+    assert data == JOB_COMMAND
+
+
+def test_job_definition_schema():
+    data = JobDefinitionSchema().load(JOB_DEFINITION_BODY)
+    assert data == JOB_DEFINITION
+
+
+def test_job_schema():
+    data = JobSchema().load(JOB_BODY)
+    assert data == JOB
+
```

Comment:
Can we also add all of these schemas to the `test_schemas_invalid_data` parameterised test?